tools/protovalidate-conformance/config.go

```diff
@@ -16,35 +16,39 @@ package main
 
 import (
 	"fmt"
+	"io"
 	"log"
 	"os"
 	"regexp"
 	"time"
 
 	flag "github.com/spf13/pflag"
+	"gopkg.in/yaml.v3"
 )
 
 type config struct {
-	suiteFilterPattern string
-	suiteFilter        *regexp.Regexp
-	caseFilterPattern  string
-	caseFilter         *regexp.Regexp
-	suiteTimeout       time.Duration
-	verbose            bool
-	strict             bool
-	strictMessage      bool
-	strictError        bool
-	proto              bool
-	json               bool
-	print              bool
-	dump               bool
-	benchmark          int
-	cmd                string
-	args               []string
+	suiteFilterPattern  string
+	suiteFilter         *regexp.Regexp
+	caseFilterPattern   string
+	caseFilter          *regexp.Regexp
+	suiteTimeout        time.Duration
+	verbose             bool
+	strict              bool
+	strictMessage       bool
+	strictError         bool
+	proto               bool
+	json                bool
+	print               bool
+	dump                bool
+	benchmark           int
+	cmd                 string
+	args                []string
+	expectedFailureFile string
+	skippedTests        map[string][]string
```

Comment:
Can we rename this to field also to `expectedFailures` because it is build from that flag's data file

Reply:
ahhh last one 👍 [728be84](https://github.com/bufbuild/protovalidate/pull/39/commits/728be8451e69c1e67047b8e2758d5567e374f26b)